lib/awful/widget/launcher.lua

```diff
@@ -12,8 +16,10 @@ local button = require("awful.button")
 local launcher = { mt = {} }
 
 --- Create a button widget which will launch a command.
--- @param args Standard widget table arguments, plus image for the image path
--- and command for the command to run on click, or either menu to create menu.
+-- @tparam table args
+-- @tparam string args.image image path to an image
+-- @tparam string args.command command to run on click
+-- @tparam table args.menu table of items to create a menu
 -- @return A launcher widget.
```

Comment:
I'd also add the widget type for the return:
```suggestion
-- @treturn awful.widget.launcher A launcher widget.
```